web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -327,7 +327,7 @@ private String getContextPath(ServletContext servletContext) {
 		}
 		
 		// trim off initial slash if it exists
-		if (contextPath.contains("/")) {
```

Comment:
Do you mind explaining the reason behind replacing contains with indexOf?

Reply:
It's just old PR. I will update this with master :)

Reply:
@dkayiwa Fixed